usaspending_api/api_contracts/contracts/v2/reporting/agencies/overview.md

```diff
@@ -0,0 +1,119 @@
+            + `missing_tas_accounts_total`
+            + `agency_name`
+            + `obligation_difference`
+            + `recent_publication_date`
+            + `recent_publication_date_certified`
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + `page_metadata` (required, PaginationMetadata, fixed-type)
+        + `results` (required, array[AgencyData], fixed-type)
+    + Body
+
+            {
+                "page_metadata": {
+                    "page": 1,
+                    "next": 2,
+                    "previous": 0,
+                    "hasNext": false,
+                    "hasPrevious": false,
+                    "total": 2,
+                    "limit": 10
+                },
+                "results": [
+                    {
+                        "agency_name": "Department of Health and Human Services",
+                        "abbreviation": "DHHS",
+                        "agency_code": "020",
+                        "current_total_budget_authority_amount": 8361447130497.72,
+                        "recent_publication_date": "2020-01-10T11:59:21Z",
+                        "recent_publication_date_certified": false,
+                        "tas_account_discrepancies_totals": {
+                            "gtas_obligation_total": 55234,
+                            "tas_obligations_not_in_gtas_total": 343345,
+                            "missing_tas_accounts_count": 20
+                        },
+                        "obligation_difference": 436376232652.87
+                    },
+                    {
+                        "agency_name": "Department of Treasury",
+                        "abbreviation": "DOT",
+                        "agency_code": "021",
+                        "current_total_budget_authority_amount": 8361447130497.72,
+                        "recent_publication_date": null,
+                        "recent_publication_date_certified": true,
+                        "tas_account_discrepancies_totals": {
+                            "gtas_obligation_total": 66432,
+                            "tas_obligations_not_in_gtas_total": 11543,
+                            "missing_tas_accounts_count": 10
+                        },
+                        "obligation_difference": 436376232652.87
+                    }
+                ]
+            }
+
+# Data Structures
+
+## PaginationMetadata (object)
++ `page` (required, number)
++ `next` (required, number, nullable)
++ `previous` (required, number, nullable)
++ `hasNext` (required, boolean)
++ `hasPrevious` (required, boolean)
++ `total` (required, number)
++ `limit` (required, number)
+
+## TASTotals (object)
++ `gtas_obligation_total` (required, number)
++ `tas_obligations_not_in_gtas_total` (required, number)
++ `missing_tas_accounts_count` (required, number)
+
+## AgencyData (object)
++ `agency_name` (required, string)
++ `abbreviation` (required, string)
++ `agency_code` (required, string)
```

Comment:
I recall someone mentioning that not all agencies listed in the table of the main "Agency Submission Statistics" page will have a drilldown / link to a single agency page. Will that be indicated by a null `agency_code`, in which case this field should be nullable?

Reply:
Every agency has a code, otherwise no financial data could be linked to it.

Recent endpoints contain a boolean flag to notify the frontend if the agency should be linked to a profile page

Reply:
Either I was mistaken or I can't find an example of the boolean field.

One example I found provided the agency code but returns a null id for the agency_id. While this might appear confusing it makes more sense than a null agency code

![image](https://user-images.githubusercontent.com/22781949/100926733-203a4200-34a1-11eb-9aa8-0d3beb8c0597.png)